ci-operator/config/openshift-service-mesh/sail-operator/openshift-service-mesh-sail-operator-main.yaml

```diff
@@ -14,16 +14,17 @@ releases:
     release:
       architecture: arm64
       channel: stable
-      version: "4.16"
+      version: "4.17"
```

Comment:
we don't need change here. The only place is related to the remote cluster version is using the env `HYPERSHIFT_HC_RELEASE_IMAGE` when provisioning a cluster using the hypershift workflow.